src/fu-util.c

```diff
@@ -827,6 +831,9 @@ fu_util_report_history (FuUtilPrivate *priv, gchar **values, GError **error)
 			return FALSE;
 	}
 
+	/* TRANSLATORS: success message -- where the user has uploaded
+	 * success and/or failure reports to the remote server */
+	g_print ("%s\n", _("Successfully reported all history"));
```

Comment:
Maybe use `g_hash_table_size (report_map)` for number of items here?

Reply:
Good idea, done.